src/packagedcode/cocoapods.py

```diff
@@ -170,3 +211,249 @@ def parse_person(person):
         email = parsed.group('email')
 
     return name, email
+
+
+@attr.s
+class CocoapodsXcodePackage(models.Package):
+    metafiles = ('*podfile.lock',)
+    extensions = ('.lock',)
+    default_type = 'pods'
+    default_primary_language = 'Objective-C'
+    hashed_directory_path = None
+    reponame = None
+    default_web_baseurl = 'https://cocoapods.org'
+    github_specs_repo_baseurl = 'https://github.com/CocoaPods/Specs/blob/master/Specs'
+    default_cdn_baseurl='https://cdn.cocoapods.org/Specs/2/2/2'
```

Comment:
https://cdn.cocoapods.org/Specs/2/2/2 is likely not right and mirrors the same structure as in the repo